addon/helpers/no-op.js

```diff
@@ -0,0 +1,7 @@
+import { helper } from '@ember/component/helper';
+
+export function noOp() {
+  return () => {};
+}
+
+export default helper(noOp);
```

Comment:
Is this helper a leftover or from another branch? It doesn't seem to be used in this PR 👀

Reply:
it's from the first iteration of changes, but eventually removed more arguments (ex: `onFocus`, `onBlur`, etc), so was no longer needed.
Kept it as it might be useful for other updates, but don't mind removing it either

Reply:
I'd say let's add it when we need it, otherwise it's outside the scope of this PR.

Reply:
thanks!
generally, agree with that feedback, in this case I think it's just more work to remove now and add back later, since it's so small and unimportant piece, but will happily update if you think that's better

regardless, do you think this ^ should be a blocker here?

Reply:
OK, if we're definitely going to use it elsewhere and you prefer to keep it 🤷‍♂️